packages/components/package.json

```diff
@@ -52,7 +52,7 @@
   },
   "homepage": "https://github.com/LabKey/labkey-ui-components#readme",
   "dependencies": {
-    "@labkey/api": "1.18.4",
+    "@labkey/api": "1.18.4-fb-deleteprojectsinapp.0",
```

Comment:
When adding a new dependency you'll need to run `yarn install` afterwards to get the associated `yarn.lock` updates.

Reply:
Oops, forgot to add this file, thank you!